detectable/src/main/java/com/synopsys/integration/detectable/detectables/yarn/parse/YarnLockParser.java

```diff
@@ -54,21 +56,48 @@ public YarnLock parseYarnLock(final List<String> yarnLockFileAsList) {
             final String trimmedLine = line.trim();
             final int level = lineLevelParser.parseIndentLevel(line);
             if (level == 0) {
-                fuzzyIds.addAll(getFuzzyIdsFromLine(line));
+                entries.add(new YarnLockEntry(ids, resolvedVersion, dependencies));
```

Comment:
Won't this add a new YarnLockEntry with an empty list of ids, an empty String for resolvedVersion, and and empty list of dependencies on the very first entry?